### ISSUE TITLE
Reproducing results in the paper

### ISSUE THREAD

**Issue by DRSY** [ShichaoSun/ConAbsSum#1]
Hi,

I have been using this codebase to run experiment XSum. Is the provided script for XSum the one u used to produce the results reported in the paper? I can only get 46.96 Rouge-1 score.

Thanks

**Comment by ShichaoSun** [comment]
Hi,

Really thanks for your interest in my paper and code.

I also meet the same issue that the results become unstable when I try to use a different computer(with 3090 GPU) although I use the same hyperparameters with the provided script. It is interesting that I can replicate the experiments on the same computer (with 1080Ti GPU). I think this is because our method is trained based on the generated silver summary, which can be altered as the rounding of different GPU varies. 

This needs to adjust the hyperparameters to obtain a common improvement for different datasets or computer configurations.  Empirically adjust the margin value or add scaling factor to NLL loss, while other hyperparameters mostly come from the PEGASUS fine-tuning configurations file and keep fixed. The margin value or scaling factor is usually set to a small value from 0.0 to 2.0. If the model overfits quickly, reduce the learning rate with a decay rate 0.1 or 0.5, and simultaneously decrease the weight-decay using the same rate. In most cases, we can get a competitive result within 10 trials. 

This process (refer to A.2 of my paper) makes my method look trivial, but I just replicate my reported result in this way when I run my code on a different computer. 

Hope this interpretation can help you.
